### PR TITLE
Update custom-branch-settings.md

### PR DESCRIPTION
**Pull request by mirnawong1**
adding info about main branch being editable if custom branch is set. [per slack thread](https://dbt-labs.slack.com/archives/C02SRNY2EQ4/p1697060648083449)

### CODE REVIEW

File: website/docs/faqs/Environments/custom-branch-settings.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: How do I use the `Custom Branch` settings in a dbt Cloud Environment?
+title: How do I use the 'Custom Branch' settings in a dbt Cloud Environment?
 description: "Use custom code from your repository"
 sidebar_label: 'Custom Branch settings'
```
Comment (mirnawong1) [inline website/docs/faqs/Environments/custom-branch-settings.md]:
```suggestion
sidebar_label: 'Custom branch settings'
```

File: website/docs/faqs/Environments/custom-branch-settings.md [hunk 2]
```diff
@@ -15,12 +15,21 @@ To specify a custom branch:
 
 ## Development
 
-In a development environment, the default branch (commonly the `main` branch) is a read-only branch found in the IDE's connected repositories, which you can use to create development branches. Identifying a custom branch overrides this default behavior. Instead, your custom branch becomes read-only and can be used to create development branches. You will no longer be able to make commits to the custom branch from within the dbt Cloud IDE.
+In a development environment, the default branch (usually named `main`) is a read-only branch in your connected repositories, which allows you to create new branches for development based on it.
```
Comment (dave-connors-3) [inline website/docs/faqs/Environments/custom-branch-settings.md]:
```suggestion
In a development environment, the default branch (usually named `main`) is a read-only branch in your connected repositories, which allows you to create new branches for development from it.  
```

File: website/docs/faqs/Environments/custom-branch-settings.md [hunk 3]
```diff
@@ -15,12 +15,21 @@ To specify a custom branch:
 
 ## Development
 
-In a development environment, the default branch (commonly the `main` branch) is a read-only branch found in the IDE's connected repositories, which you can use to create development branches. Identifying a custom branch overrides this default behavior. Instead, your custom branch becomes read-only and can be used to create development branches. You will no longer be able to make commits to the custom branch from within the dbt Cloud IDE.
+In a development environment, the default branch (usually named `main`) is a read-only branch in your connected repositories, which allows you to create new branches for development based on it.  
 
-For example, you can use the `develop` branch of a connected repository. Edit an environment, select  **Only run on a custom branch** in **General settings** , enter **develop** as the name of your custom branch.
+Specifying a **Custom branch** overrides the default behavior. It makes the custom branch 'read-only' and enables you to create new development branches based on it. This also means you can't edit this custom branch directly.
```
Comment (dave-connors-3) [inline website/docs/faqs/Environments/custom-branch-settings.md]:
```suggestion
Specifying a **Custom branch** overrides the default behavior. It makes the custom branch 'read-only' and enables you to create new development branches from it. This also means you can't edit this custom branch directly.
```

**Review by dave-connors-3 — APPROVED** [review]

**Comment by mirnawong1** [comment]
awesome, thank you soo much @dave-connors-3 and applied your changes!

**Pull request merged**